### ISSUE TITLE
Remove http query params from destination file name `release-2.x` branch

### ISSUE THREAD

**Issue by dmikusa** [paketo-buildpacks/libpak#287]
## Describe the Enhancement

See #273

## Motivation

See https://github.com/paketo-buildpacks/libpak/pull/273 

## Possible Solution

Port to release-2.x, but we also need to consider how this impacts offline usage.

**Comment by pranavek** [comment]
@dmikusa @anthonydahanne The changes on v1 are reverted. Shall we change the possible solution? If you could help me understand how you want to proceed with this, I can work on this.

**Comment by anthonydahanne** [comment]
ah, that's a good point.
now since we're talking v2, I think it could apply here - wdyt @dmikusa ?
cc @pivotal-david-osullivan

**Comment by dmikusa** [comment]
Yes, I agree. You can resubmit the PR against the v2 branch. That will get things moving. 

I do want to have another think through all this, to contemplate the offline use case. That's what was overlooked previously.

**Comment by loewenstein** [comment]
The title seems to be misleading... If I followed all the correct links, then the original PR #273 was reverted on `main`, right? I guess best would be to close this issue and instead reopen #274 - no?
